packages/core/src/common/logger-protocol.ts

```diff
@@ -37,3 +37,65 @@ export interface ILogLevelChangedEvent {
 export interface ILoggerClient {
     onLogLevelChanged(event: ILogLevelChangedEvent): void;
 }
+
+export const rootLoggerName = 'root';
```

Comment:
Dependencies supposed to be `ILogger` -> `ILoggerServer`, but because implementations of `ILoggerServer` using `LogLevel` and  `rootLoggerName` they were introducing a cycle, the only commonplace which I found was a protocol.

Reply:
ok